docs/pages/blog/turbo-1-5-0.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Turborepo 1.5
+date: 2022/09/19
+description: Turborepo 1.5 brings the Monorepo Handbook, drops the 'run' command, improves pruning, and much more!
+tag: web development
+ogImage: /images/blog/turbo-1-4-0/og-image.png
+---
+
+# Turborepo 1.5
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Monday, September 19th, 2022</div>
+
+<Authors authors={[ 'mattpocock', 'gsoltis', 'nathanhammond', 'tknickman', 'anthonyshewdev', 'jaredpalmer', 'mehulkar' ]} />
+
+Turborepo 1.5 is a **huge leap forward for our documentation and DX**, as well as bringing big improvements to `turbo prune`:
+
+- [**The Monorepo Handbook**](#the-monorepo-handbook): We've built the missing manual for your monorepo - a guide on workspaces, code sharing, integrating common tools and much more.
+- [**No need to run**](#run-no-longer-needed): `turbo run <task>` can now be shortened to `turbo <task>`
+- [**Prune now supported on pnpm, yarn and npm**](#prune-now-supported-on-npm-pnpm-and-yarn): Pruning your monorepo is now supported in all major package managers.
+- [**Improved environment variables in `turbo.json`**](#environment-variables-in-turbojson): Environment variables are now first-class citizens in your Turborepo pipeline configuration.
+- [**Changes to `package.json` hashing**](#changes-to-packagejson-hashing): We've improved how we hash `package.json` when running tasks.
+
+Update today by running `npm install turbo@latest`.
+
+## The Monorepo Handbook
+
+Setting up a monorepo for the first time often means navigating a lot of new concepts. You'll need to understand workspaces, package installation, sharing code and dependency management - and a lot more.
+
+This often meant that folks who wanted to set up a monorepo from scratch had to piece information together from different documentation sites. First `pnpm`, then `tsup`, then back to `changesets`, then back to Turborepo for dessert.
+
+We want to fill this gap with [the Monorepo Handbook](/docs/handbook). We've built guides on how to integrate all the tools you'll need to make ship happen with your monorepo, including guides on:
+
+- Installing Packages
+- Linting
+- Development Tasks
+- Building Apps
+- Publishing Packages
+
+{/* TODO - link these */}
+
+## `run` no longer needed
+
+You can now run tasks with the Turborepo CLI using `turbo <task>`.
+
+```diff
+- turbo run build
++ turbo build
+
+
+- turbo run lint build test
++ turbo lint build test
+```
+
+If your task name conflicts with a built-in `turbo` subcommand, we'll run our subcommand instead. That means you shouldn't name your tasks things like `prune`, `run`, or `login` - since those are built-in subcommands.
+
+`turbo run <task>` will continue to work, and there are no plans to deprecate it.
+
```

Comment:
Now that we've moved to cobra, flag parsing is now more relaxed. 

We actually need to also update this section since spaces are now allowed. You don't need `--flag="foo"` but can safely do `--flag foo` or `--flag "foo"` now. We might want to call this out as a separate bullet. https://turborepo.org/docs/reference/command-line-reference#option-syntax

Reply:
Done, in separate PR